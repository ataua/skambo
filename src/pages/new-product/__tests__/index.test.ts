import { createRenderer } from 'react-dom/test-utils';
import NewProduct from '../';
