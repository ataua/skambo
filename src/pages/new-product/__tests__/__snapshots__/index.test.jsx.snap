// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewProduct Should render NewProduct 1`] = `
<div
  className="sc-fujyUd kgJPlS"
  style={
    Object {
      "marginTop": 80,
    }
  }
>
  <h1>
    Novo Produto
  </h1>
  <a />
  <form
    className="ui form"
    onSubmit={[Function]}
  >
    <div
      className="required field"
    >
      <label
        htmlFor="name"
      >
        Produto
      </label>
      <input
        id="name"
        name="name"
        placeholder="Nome do Produto"
        type="text"
      />
    </div>
    <div
      className="required field"
    >
      <label
        htmlFor="category"
      >
        Categoria
      </label>
      <select
        defaultValue="Outros"
        id="category"
        name="category"
        placeholder="Escolha uma categoria"
      >
        <option
          value="Eletronicos"
        >
          Eletrônicos
        </option>
        <option
          value="Escritorio"
        >
          Escritório
        </option>
        <option
          value="Esportes"
        >
          Esportes
        </option>
        <option
          value="Instrumentos_Musicais"
        >
          Instrumentos Musicais
        </option>
        <option
          value="Jogos"
        >
          Jogos
        </option>
        <option
          value="Lazer"
        >
          Lazer
        </option>
        <option
          value="Livros"
        >
          Livros
        </option>
        <option
          value="Moveis"
        >
          Móveis
        </option>
        <option
          value="Saude"
        >
          Saúde
        </option>
        <option
          value="Servicos"
        >
          Serviços
        </option>
        <option
          value="Vestuario"
        >
          Vestuário
        </option>
        <option
          value="Outros"
        >
          Outros
        </option>
      </select>
    </div>
    <div
      className="required field"
    >
      <label
        htmlFor="images"
      >
        Imagens
      </label>
      <input
        accept="image/x-png,image/gif,image/jpeg"
        id="images"
        multiple={true}
        name="images"
        onChange={[Function]}
        placeholder="Inserir imagem"
        type="file"
      />
    </div>
    <div
      className="field"
    >
      <label
        htmlFor="usability"
      >
        Estado de conservação
      </label>
      <input
        id="usability"
        max={5}
        min={1}
        name="usability"
        onChange={[Function]}
        type="range"
      />
      <div>
        selecione
      </div>
    </div>
    <div
      className="required field"
    >
      <label
        htmlFor="description"
      >
        Detalhes
      </label>
      <textarea
        id="description"
        name="description"
        placeholder="Detalhes como possíveis sinais de uso,  manchas, partes faltantes ou estragadas, para que os  visitantes saibam o que esperar do seu produto."
        rows={4}
      />
    </div>
    <div
      className="required field"
    >
      <label
        htmlFor="value"
      >
        Valor de referência para Skambo
      </label>
      <input
        id="value"
        maxLength={11}
        name="value"
        onChange={[Function]}
        placeholder="1,00"
        type="tel"
        value="0"
      />
    </div>
    <div
      className="field"
    >
      <label
        htmlFor="interests"
      >
        Interesses 
        <em
          style={
            Object {
              "color": "grey",
            }
          }
        >
          opcional
        </em>
      </label>
      <input
        id="interests"
        name="interests"
        placeholder="Interesses para troca, separados por vírgula"
        type="text"
      />
    </div>
    <div
      className="field"
    >
      <div
        className="sc-jSFkmK boYQcN"
      >
        <button
          className="sc-bdnylx sc-gtssRu fHFSIS ctxtJV"
          onClick={[Function]}
          type="reset"
        >
          Limpar
        </button>
        <button
          className="sc-bdnylx sc-dlnjPT fHFSIS gWCxVm"
        >
          Cadastrar
        </button>
      </div>
    </div>
  </form>
</div>
`;
